spec/system/admin/bulk_url_upload_spec.rb

```diff
@@ -37,94 +78,42 @@
   describe 'bulk uploading a file of URLs' do
     include_context 'log in super admin'
```

Comment:
We can leave this as it was, but we also have the opportunity to wrap all the specs in a single context to avoid duplication:
```ruby
context `when a super admin is logged in` do
  include_context 'log in super admin'
  
  describe 'bulk uploading a file of URLs' do...
  ```

Reply:
Changed.

Reply:
Changed.